### DISCUSSION TITLE
Meeting minutes for 27.07.2023

### DISCUSSION THREAD

**Discussion by oxisto** [Fraunhofer-AISEC/cpg#1253] [Dev Meetings]
### Agenda

1. Open PRs
2. `v8` development cycle started

- Breaking Name changes: E.g. DeclaredReferenceExpression what else?

3. Automatic "release" on maven central via CI
4. Github runner: What do we have, what do we want, what can we get?
5. Split out Go code into separate repo?

### Discussed PRs

- [x] #1199 
- [ ] #1249 
...

**Comment by konradweiss** [comment]
## Open PRs

#1249  Translation logic is moved to kotlin code to allow for better debugging.

## `v8` development cycle started
- Rename DeclaredReferenceExpression
- Remove TypeListeners
- AssignExpressions in all frontends

## Automatic "release" on maven central via CI
- New publish plugin runs until closing of repository. Manual release is still required to allow for quality assurance of the artifact.

## Github runner
- Previous runners made builds too slow
- Larger runners need a billable per minute contract over a paid account
- We currently use a custom runner in a cloud VM

## Split out Go code into separate repo?
- To not build the go part every time, a C wrapper around it can be extracted into a new library.
